src/rprblender/nodes/node_parser.py

```diff
@@ -273,7 +293,9 @@ def final_export(self):
         """
 
         log("export", self.material, self.node, self.socket_out, self.group_nodes)
-        if isinstance(self.rpr_context, RPRContextHybrid):
+        if self.node.mute:
+            return self.export_muted()
```

Comment:
Fails with
...
  File "D:\Work\RPR\blender_280\RadeonProRenderBlenderAddon\src\bindings\pyrpr\.build\pyrprwrap.py", line 1393, in ShapeSetMaterial
    return lib.rprShapeSetMaterial(shape._get_handle() if shape else ffi.NULL, node._get_handle() if node else ffi.NULL)
AttributeError: 'NodeItem' object has no attribute '_get_handle'
if entry shader node is muted, for example Mix Shader node connected to Output.